apps/backoffice-v2/src/lib/blocks/hooks/useUbosBlock/useUbosBlock.tsx

```diff
@@ -1,30 +1,19 @@
 import { useMemo } from 'react';
 import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { WarningFilledSvg } from '@/common/components/atoms/icons';
 
-export const useUbosBlock = ubos => {
-  return useMemo(() => {
-    if (!Array.isArray(ubos) || !ubos?.length) {
-      return [];
-    }
+type Ubo = {
+  name?: string;
+  percentage?: number;
+  type?: string;
+  level: number;
```

Comment:
Are all these truly optional are is this supposed to be a union? i.e. `{ level: number; } | { name: string; percentage: number; type: string; level: number; }`

Reply:
We don't normalize the data from UBO right now so I want to be on the safe side. I see that I missed one with the `level` property